resources/packs/gcp/gcp.lr

```diff
@@ -46,7 +46,8 @@ gcp.project @defaults("name") {
   recommendations() []gcp.recommendation
   // GCP Compute Resources for the Project
   // compute() gcp.compute
-  clusters() []gcp.project.cluster
+  // GCP GKE resources
+  gke() gcp.project.gkeService
```

Comment:
I think it is a great call. Do we need an alias for it?

Reply:
I think not since this was released last week and we are not using it just yet.

Reply:
Sounds good to me, just coordinate with Tim, that we update the release notes.